ravenframework/MessageHandler.py

```diff
@@ -66,10 +58,6 @@ class NoMoreSamplesNeeded(GeneratorExit):
 
 TL;DR: BaseClasses/MessageUser is a superclass that gives access to hooks to the simulation's MessageHandler
 instance, while the MessageHandler is an output stream control tool.
-
-In an effort to make the MH more flexible, we insert getMessageHandler into the python "builtins" module.
-This means that any time after this module (MessageHandler) is imported, you can use
-"getMessageHandler(name='default')" to retrieve a particular message handler as identified by "name".
```

Comment:
this was an intentional change requested by a previous PR; is there a critical reason to change it?

Reply:
No. I disagree with the previous implementation, but since it worked, I will put it back.